mono/mini/mini-runtime.c

```diff
@@ -4691,9 +4693,21 @@ register_icalls (void)
 
 MonoJitStats mono_jit_stats = {0};
 
+void
+mono_stats_lock (void)
```

Comment:
I would prefer to call this `mono_runtime_stats_*`. In general, you should group functions under a label like that since C doesn't have namespaces.